src/dataExplorer/components/SaveAsNotebookForm.tsx

```diff
@@ -41,7 +41,7 @@ interface Props {
 const SaveAsNotebookForm: FC<Props> = ({dismiss}) => {
   const [notebookName, setNotebookName] = useState('')
   const orgID = useSelector(getOrg).id
-  const {draftQueries, autoRefresh, timeRange} = useSelector(
+  const {draftQueries, autoRefresh, timeRange, activeQueryIndex} = useSelector(
```

Comment:
Just curious, what does `activeQueryIndex` do?  My understanding is that there can be many queries but only one is active when the user actively working on it, and in this case, the user is saving a specific query from data explorer to notebook